### PR TITLE
graph-tool 2.27

### PR DESCRIPTION
**Pull request by count0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/graph-tool.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class GraphTool < Formula
 
   desc "Efficient network analysis for Python 3"
   homepage "https://graph-tool.skewed.de/"
-  url "https://downloads.skewed.de/graph-tool/graph-tool-2.26.tar.bz2"
-  sha256 "df6273dc5ef327a0eaf1ef1c46751fce4c0b7573880944e544287b85a068f770"
-  revision 2
+  url "https://downloads.skewed.de/graph-tool/graph-tool-2.27.tar.bz2"
+  sha256 "4740c69720dfbebf8fb3e77057b3e6a257ccf0432cdaf7345f873247390e4313"
+  revision 1
```
Comment (ilovezfs) [inline Formula/graph-tool.rb]: revision counting starts at 0 which is left unwritten

Comment (count0) [reply Formula/graph-tool.rb]: Fixed...

**Comment by ilovezfs** [comment]
looks like there's an update for python-dateutil
```
  resource "python-dateutil" do
    url "https://files.pythonhosted.org/packages/a0/b0/a4e3241d2dee665fea11baec21389aec6886655cd4db7647ddf96c3fad15/python-dateutil-2.7.3.tar.gz"
    sha256 "e27001de32f627c22380a688bcc43ce83504a7bc5da472209b4c70f02829f0b8"
  end
```

**Comment by fxcoudert** [comment]
Thanks @count0 for the pull request!

**Pull request closed without merging**